clients/admin-ui/src/pages/setup.tsx

```diff
@@ -0,0 +1,55 @@
+import { Button, Container, Heading, Image, Stack } from "@fidesui/react";
+import type { NextPage } from "next";
+import React from "react";
+
+const Setup: NextPage = () => (
+  <Stack>
+    <Container
+      height="35vh"
+      maxW="100vw"
+      padding="0px"
+      bg="white"
+      color="white"
+    >
+      <Image
```

Comment:
hmm why is this here?

Reply:
There's a placeholder image for the data map in the designs. I am waiting for that src.